src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -32,6 +32,11 @@
       "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
   public static final String SQL_LOCAL_USER = "root";
   public static final String SQL_LOCAL_PASSWORD = "";
+
+  private static final String dataBaseName = "Mintern";
```

Comment:
This should also be a constant, so it should be named something like DATABASE_NAME

Reply:
Done!